proto_converter.go

```diff
@@ -0,0 +1,186 @@
+package lightstep
+
+import (
+	"fmt"
+	google_protobuf "github.com/golang/protobuf/ptypes/timestamp"
+	cpb "github.com/lightstep/lightstep-tracer-go/collectorpb"
+	ot "github.com/opentracing/opentracing-go"
+	"reflect"
+	"time"
+)
+
+type protoConverter struct {
+	verbose        bool
+	maxLogKeyLen   int // see GrpcOptions.MaxLogKeyLen
+	maxLogValueLen int // see GrpcOptions.MaxLogValueLen
+}
+
+func newProtoConverter(options Options) *protoConverter {
```

Comment:
This entire file makes me so happy 👍  to _more_ organization!

Reply:
😄